lib/components/app/app-menu.tsx

```diff
@@ -204,13 +203,11 @@ class AppMenu extends Component<
             label:
               activeLocale === locale ? (
                 <>
-                  <strong
-                    aria-label={intl.formatMessage({
-                      id: 'components.SubNav.activeLanguage'
-                    })}
-                  >
-                    {configLanguages[locale].name}
-                  </strong>
+                  {/* // Text is screen-reader only */}
+                  <span style={{ display: 'none' }}>
```

Comment:
Revert this change in favor of #751.

Reply:
There is no room for the language selector on mobile. It should stay in the app menu

Reply:
Okay, thanks for the reminder; however `display: none` also hides contents from screen readers, so either hide the text visually using small dimensions, or use `aria-selected`, or `aria-label`.